### ISSUE TITLE
Automatically block known SR scammers

### ISSUE THREAD

**Issue by Jamyn** [waylaidwanderer/Mist#20]
Feature idea: So, having some sort of in-chat notification for SR scammers is useful. Someone may have been legit and then decided to scam later, but they're already in your friends list for example.

But I think having an (optional) configuration item (checkbox?) that allows them to auto-ignore, auto-block known scammers would be nice.

So on the "Incoming friend request" event, we could probably check their rep, and drop the request without even showing it to the end-user if it's from a known scammer.

**Comment by waylaidwanderer** [comment]
Oh, and maybe if not set to auto-ignore/block, pop up a messagebox when a scammer adds you and ask if you want to ignore/block or add them back?

**Comment by Jamyn** [comment]
Nice idea

**Comment by ghost** [comment]
True. I'm willing to deal with a scammer if they deal with non reversible terms (Liberty reserve only, they go first, they pay 3-5$ fee). 
Sent from my Verizon Wireless BlackBerry

-----Original Message-----
